Aztec/Classes/TextKit/TextStorage.swift

```diff
@@ -171,53 +197,52 @@ open class TextStorage: NSTextStorage {
     /// - Returns: the preprocessed string.
     ///
     fileprivate func preprocessAttachmentsForInsertion(_ attributedString: NSAttributedString) -> NSAttributedString {
-        
+        assert(attachmentsDelegate != nil)
```

Comment:
Just to add some info on this, @jleandroperez had a pretty nice idea for improving this mechanism in the future.

I'd encourage us as a team to review the architecture of all our delegates as an improvement.  Maybe after closed beta.